website/www/site/i18n/home/hero/en.yaml

```diff
@@ -13,8 +13,8 @@
 - id: home-hero-title
   translation: Introducing Apache Beam
 - id: home-hero-heading
-  translation: An advanced unified programming model
+  translation: The Unified Apache Beam Model
```

Comment:
I'd recommend sentence case here: "The unified Apache Beam model".

Reply:
ditto